package.json

```diff
@@ -19,6 +19,7 @@
     "node": ">=18.0.0"
   },
   "dependencies": {
+    "@pinecone-database/pinecone": "^2.2.2",
```

Comment:
Is the Issue resolved now ?

Reply:
Adding the @pinecone-database/pinecone dependency! My branch was breaking because of that missing dep.